chall02/tdawson.c

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+char *hv_rgb2hex(int r, int g, int b)
+{
+	char *hex_str;
+
+	if (!(hex_str = malloc(8)))
+		return NULL;
+	unsigned int hex = ((r & 0xff) << 16) + ((g & 0xff) << 8) + (b & 0xff);
```

Comment:
wuw :) bitwise!

where did you get it?

Reply:
I found it when doing research for FDF but I can't remember where, probably from Stack Overflow (where else 😁)

Reply:
I found it when doing research for my FDF but I can't remember where, probably Stack Overflow (where else 😁)

Reply:
Stack Overflow :) the developer toolbox!